### PR TITLE
Update homepage of 18 Casks

### PR DESCRIPTION
**Pull request by miccal**
Update homepage of 18 Casks.

Ping @victorpopkov

### CODE REVIEW

File: Casks/frizzix.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@
   appcast 'http://frizzix.de/downloads/FrizzixUpdate.xml',
           checkpoint: '1e452fb9339f33941d46d3195b9d8b4a2e90c2ccd63d42fcbf303151247175f4'
   name 'Frizzix'
-  homepage 'http://mac.frizzix.de/'
+  homepage 'http://mac.frizzix.de/de/'
```
Comment (victorpopkov) [inline Casks/frizzix.rb]: This should be left as is since is's used for localization purposes.

File: Casks/freeplane.rb [hunk 2]
```diff
@@ -6,7 +6,7 @@
   appcast 'https://sourceforge.net/projects/freeplane/rss?path=/freeplane%20stable',
           checkpoint: '60fc48e1bc2033f5862ffb60ff689372bc9409406f254a39bb966ab833ee719a'
   name 'Freeplane'
-  homepage 'http://freeplane.sourceforge.net/'
+  homepage 'http://freeplane.sourceforge.net/wiki/index.php/Main_Page'
```
Comment (victorpopkov) [inline Casks/freeplane.rb]: I would suggest to not change this and leave all the SourceForge links as they are.

File: Casks/eye-one-profiler.rb [hunk 3]
```diff
@@ -4,7 +4,7 @@
 
   url 'http://www.xrite.com/downloader.aspx?FileID=1455&Type=M&returnurl=%2fi1profiler-i1publish%2fsupport%2fd1455'
   name 'i1Profiler'
-  homepage 'http://www.xrite.com/i1profiler-i1publish/support/d1455'
+  homepage 'http://www.xrite.com/categories/formulation-and-quality-assurance-software/i1Profiler'
```
Comment (victorpopkov) [inline Casks/eye-one-profiler.rb]: This cask should be considered for removal. The download URL is dead.

**Review by victorpopkov — CHANGES_REQUESTED** [review]

**Comment by miccal** [comment]
Thanks @victorpopkov - I will just revert the changes to those Casks that should be removed, and I will submit a separate PR later to remove them.

**Review by victorpopkov — APPROVED** [review]

**Pull request merged**